frameworks/kafka/docs/upgrade.md

```diff
@@ -106,4 +110,43 @@ Since protocol and log versions have been updated with new protocol and log form
 **Note**: Default settings for protocol and log versions are overwritten with these customized options (steps 1 through 3), even though  their values are same as the defaults of the new package version `2.0.0-0.11.0.0` . Pay attention to these customized options for further package upgrades since they will be preserved unless overwritten explicitly.
 
     
-    
+##  Upgrade in Strict Mode
+
+If you are upgrading to the DC/OS Apache Kafka package `2.0.0-0.11.0.0`  on a DC/OS cluster running in [strict security mode](https://docs.mesosphere.com/1.9/security/#security-modes), you must specify service account details during the package update process. Service account credentials enable your service to authenticate to a DC/OS cluster in strict mode.
+
+
+You can provide service account details in a JSON options file or via the DC/OS GUI.
+
+
+If you are **installing from scratch** in strict mode, add the following parameter to your options JSON file in order to  provide your service account details.
+
+
+    $ cat options.json
+    {
+        "service": {
+            "service_account": "this_is_your_service_account_id",
+            "service_account_secret": "this_is_your_sa_secret_path"
+        }
+    }
+
+    $ docs package install  kafka --options=options.json ` --package-version=2.0.0-0.11.0.0
+
+**Note:** The syntax for specifying service account details has changed from the previous version. The `principal` and `secret_name` parameters have changed to `service_account` and `service_account_secret`. If you are upgrading your service from package version `1.1.16-0.10.1.0-beta`, you will need to specify service account details in a JSON options file.
+
```

Comment:
Suggest a header + edits
### Upgrade to Kafka `2.0.0-0.11.0.0`
"from `1.1.16-0.10.1.0-beta` to version `2.0.0-0.11.0.0`,"